### DISCUSSION TITLE
How do I get the non-basis compressed Matterport3D GLB files

### DISCUSSION THREAD

**Discussion by yukaryote** [facebookresearch/habitat-lab#1828] [Q&A]
In [this tutorial on editing assets in Blender](https://aihabitat.org/tutorial/editing_in_blender/), the author says to use non-basis-compressed GLB files from Matterport3D. I assume I don't have these non-basis-compressed GLB files from my Matterport3D download since my MP3D GLB files won't load in Blender. How do I get these GLB files? I've emailed the Matterport3D Google group with no avail.

**Comment by yunho-c** [comment]
For HM3D dataset, appending `_full` suffix to the `--uuid` of the dataset download script did the trick. For example: `python -m habitat_sim.utils.datasets_download --username <api-token-id> --password <api-token-secret> --uids hm3d_full`.

After running this, I was able to check the addition of uncompressed `<scene_id>.glb` file along with basis-compressed `<scene_id>.basis.glb` under `habitat-sim/data/scene_datasets/hm3d/<train/val/minival>/<scene_id>`.

**Reply by yukaryote** [reply]
Cool! I'll try this if I need the dataset again at some point.

If you're curious, I did end up getting non-basis compressed MP3D GLB files in a hacky way. First I went to this [website](https://aspis.cmpt.sfu.ca/scene-toolkit/scans/matterport3d/houses) which hosts all the non-basis compressed GLB files. If you click on one of the models there, you go to a model viewer UI which has a button to "exportModel", but that button doesn't seem to work. So you have to do some "inspect element" shenanigans to download the GLB file using JavaScript.
